app/assets/stylesheets/application.scss

```diff
@@ -12,10 +12,12 @@
  *
  */
 @import "bootstrap-sprockets";
+@import "bootstrap";
```

Comment:
This can go back to after bootstrap-overrides.  It was moved here to support code in bootstrap-variables that's now in layout.scss.

bootstrap-overrides would be where you put something like `$brand-info: #abcdef`, if you wanted to override bootstrap's default `info` color:  `$brand-info:            #5bc0de !default;`.  So it needs to be imported before bootstrap.